### DISCUSSION TITLE
omezyný přístup

### DISCUSSION THREAD

**Discussion by Pete5s** [community/community#147300] [Discussions]
### Select Topic Area

Question

### Body

Dobrý den, nemohu sdílet projekt v inteliJ na github z důvodu omezení přístupu? Potřeboval bych prosím poradit co s tím dělat. Děkuji

**Comment by mecodeatlas** [comment]
Hi @Pete5s, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
